configCivcraft.yml

```diff
@@ -4985,7 +4986,7 @@ recipes:
     output:
       aether:
         material: GOLD_NUGGET
-        amount: 10
+        amount: 3
```

Comment:
Change the xp recipes, not the pylon recipes. The smaller you make those numbers, the less noticeable and controllable the output gets, because it's always rounded to a full item.